Apps/ValidationTests/Scripts/text_rendering_test.js

```diff
@@ -0,0 +1,139 @@
+
+function CreateInputHandling(scene) {
+    var inputManager = new InputManager();
+    var priorX = inputManager.pointerX;
+    var priorY = inputManager.pointerY;
+    var x = 0;
+    var y = 0;
+    scene.onBeforeRenderObservable.add(function () {
+        x = inputManager.pointerX;
+        y = inputManager.pointerY;
+
+        if (inputManager.isPointerDown) {
+            scene.activeCamera.alpha += 0.01 * (priorX - x);
+            scene.activeCamera.beta += 0.01 * (priorY - y);
+        }
+
+        priorX = x;
+        priorY = y;
+    });
+}
+
+console.log("Starting");
+
+var engine = new BABYLON.NativeEngine();
+var scene = new BABYLON.Scene(engine);
+
+scene.createDefaultCamera(true);
+CreateInputHandling(scene);
+scene.createDefaultLight(true);
+
+
+engine.updateDynamicTexture = function (texture, canvas, invertY, premulAlpha, format) {
+    if (premulAlpha === void 0) { premulAlpha = false; }
+    var webGLTexture = texture._hardwareTexture.underlyingResource;
+    this._native.copyTexture(webGLTexture, canvas.getCanvasTexture());
+    texture.isReady = true;
+};
+
+console.log("Reading font");
+var textureGround;
+var font = "bold 44px monospace";
+BABYLON.Tools.LoadFile("https://raw.githubusercontent.com/CedricGuillemet/dump/master/droidsans.ttf", (data) => {
+    _native.NativeCanvas.loadTTF("droidsans", data);
```

Comment:
I'm not sure if this is on the radar, but should we be exposing this through a [`FontFace`](https://developer.mozilla.org/en-US/docs/Web/API/FontFace) polyfill? Then the calling code would just do:
```ts
const fontFace = new FontFace('droidsans', 'url(https://raw.githubusercontent.com/CedricGuillemet/dump/master/droidsans.ttf)');
fontFace.load().then(() => {
    ...
    const textBlock = new BABYLON.GUI.TextBlock();
    textBlock.fontFamily = 'droidsans';
    ...
});
```